hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/TestDefaultStringifier.java

```diff
@@ -107,6 +108,12 @@ public void testStoreLoadArray() throws IOException {
     Integer[] array = new Integer[] {1,2,3,4,5};
 
 
+    try {
```

Comment:
use our `LambdaTestUtils.intercept()` here; it's a reimplementation of scalatest's and straightforward to use it.

Reply:
we have updated the code, please review.